package-lock.json

```diff
@@ -46,7 +47,11 @@
         "rc-slider": "^10.1.0",
         "react": "^18.2.0",
         "react-bs-notifier": "^7.0.0",
+<<<<<<< HEAD
+        "react-datetime-picker": "^4.2.0",
+=======
         "react-date-picker": "^9.2.0",
+>>>>>>> 64ae1ccadfc2954f64fe08e4ccad7ee35877ff69
```

Comment:
There's a merge conflict here that was committed as-is.
Hopefully when you run the `npm remove react-date-picker`  to remove the package this will be resolved, but please do check that it is indeed resolved.
You shouldn't find anything searching for  `<<<<<<< HEAD` or `>>>>>>>`